client/src/components/controls/LineChart.vue

```diff
@@ -85,95 +84,92 @@ export default {
   methods: {
     initialize() {
       d3.select(this.$el)
-        .select("svg")
+        .select('svg')
         .remove();
-      var width = this.$el.clientWidth;
-      var height = this.$el.clientHeight;
-      var x = d3
+      let tooltipTimeoutHandle = null;
+      const tooltip = d3
+        .select(this.$el)
+        .append('div')
+        .attr('class', 'tooltip')
+        .style('display', 'none');
+      const width = this.$el.clientWidth;
+      const height = this.$el.clientHeight;
+      const x = d3
         .scaleLinear()
         .domain([this.startFrame, this.endFrame])
         .range([0, width]);
       this.x = x;
-      var max = d3.max(this.lineData, datum => d3.max(datum.values, d => d[1]));
-      var y = d3
+      const max = d3.max(this.lineData, (datum) => d3.max(datum.values, (d) => d[1]));
+      const y = d3
         .scaleLinear()
         .domain([0, Math.max(max + max * 0.2, 2)])
         .range([height, 0]);
 
-      var line = d3
+      const line = d3
         .line()
         .curve(d3.curveStepAfter)
-        .x(d => x(d[0]))
-        .y(d => y(d[1]));
+        .x((d) => x(d[0]))
+        .y((d) => y(d[1]));
       this.line = line;
 
-      var svg = d3
+      const svg = d3
         .select(this.$el)
-        .append("svg")
-        .style("display", "block")
-        .attr("width", width)
-        .attr("height", height)
-        .append("g")
-        .attr("transform", `translate(0,-1)`);
+        .append('svg')
+        .style('display', 'block')
+        .attr('width', width)
+        .attr('height', height)
+        .append('g')
+        .attr('transform', 'translate(0,-1)');
 
-      var axis = d3.axisRight(y).tickSize(width);
+      const axis = d3.axisRight(y).tickSize(width);
       this.axis = axis;
       svg
-        .append("g")
-        .attr("class", "axis-y")
+        .append('g')
+        .attr('class', 'axis-y')
         .call(axis)
```

Comment:
`this.axis` axis within the current filed is only referenced a few times and I don't think `this.axis` is referenced at all.  Not entirely sure what it is being used for?
Timeline.vue has a `this.axis` assignment in the mounted function, but it's` this.axis` is used across multiple methods.